hooks/001-extra-packages.chroot

```diff
@@ -258,3 +260,11 @@ apt-get download snapd
 dpkg --fsys-tarfile snapd_*.deb |
        tar xf - ./usr/lib/systemd/system-generators/snapd-generator
 rm snapd_*.deb
+
+# Copy nvidia udev rules and dependencies (sources for this come from
+# Canonical, not from Nvidia, see
+# https://github.com/canonical/nvidia-graphics-drivers).
+nvidia_common_pkg=nvidia-kernel-common-550
```

Comment:
Is this something we need to keep maintained/updated or should 71-nvidia.rules stay relatively intact per LTS

Reply:
It will have the usual updates from the deb if any. If you are thinking about a newer nvidia driver release, I do not think it happens too often.